backend/core/src/main/resources/application-staging.properties

```diff
@@ -12,9 +12,15 @@ amazon.bucketName = ${AWS_BUCKET}
 # Database PostgresSQL
 #-------------------------
 spring.datasource.driver-class-name=org.postgresql.Driver
-spring.datasource.username=postgres
-spring.datasource.password=admin
-spring.datasource.url=jdbc:postgresql://localhost:5432/feed_grabber
+spring.datasource.username=${DB_USERNAME}
+spring.datasource.password=${DB_PASSWORD}
+# spring.datasource.url=jdbc:postgresql://localhost:5432/feed_grabber
+spring.datasource.url=jdbc:postgresql://core-db:5432/feed_grabber
```

Comment:
I have a question about these properties. On the local machine, we have another database host (localhost). Should we move the database host to separate env variable? Like this:
`spring.datasource.url=jdbc:postgresql://${DB_HOST}:5432/feed_grabber`